src/user.ts

```diff
@@ -0,0 +1,57 @@
+import { createInterface } from "readline";
+import { GameState } from "./gamestate";
+import { UTF16_CODE_OF_LETTER_A, Coordinates } from "./utils";
+
+export function getBoardCoordinatesFromUserInput(
+  userCoordinates: string
+): Coordinates {
+  let x: number, y: number;
+  if (userCoordinates.length < 2) {
```

Comment:
What if the board is larger than 9x9 ? Stg like "K11" will be rejected although it's valid 😬